packages/core/src/modal/modal.element.ts

```diff
@@ -88,28 +104,26 @@ export class CdsModal extends CdsInternalOverlay {
   render() {
     return html`
       ${this.backdropTemplate}
-      <div cds-layout="horizontal p:md p@md:xl align:center">
-        <div class="modal-dialog private-host" tabindex="-1">
-          <div cds-layout="display:screen-reader-only">${this.i18n.contentStart}</div>
-          <div class="modal-content" cds-layout="vertical gap:md gap@md:lg align:stretch">
-            <div cds-layout="horizontal gap:md wrap:none align:vertical-center p-x:lg p-t:lg">
-              <div>
-                <slot name="modal-header"></slot>
-              </div>
-              <div cds-layout="align:right">
-                <slot name="modal-header-actions"></slot>
-              </div>
-              <slot name="close-button"></slot>
+      <div class="modal-dialog private-host" tabindex="-1" cds-layout="m:md m@md:xl">
```

Comment:
We had an unnecessary wrapper around the modal. It's possible it had utility in a previous iteration. But I have verified that it was vestigial after the overlay re-factor.